girder/utility/install.py

```diff
@@ -78,9 +78,10 @@ def _getPluginBuildArgs(buildAll, plugins):
     plugins = list(plugin_utilities.getToposortedPlugins(plugins, ignoreMissing=True))
 
     # include static-only dependencies that are not in the runtime load set
-    staticPlugins = plugin_utilities.getToposortedPlugins(
-        plugins, ignoreMissing=True, keys=('dependencies', 'staticWebDependencies'))
-    staticPlugins = [p for p in staticPlugins if p not in plugins]
+    # staticPlugins = plugin_utilities.getToposortedPlugins(
+    #     plugins, ignoreMissing=True, keys=('dependencies', 'staticWebDependencies'))
+    # staticPlugins = [p for p in staticPlugins if p not in plugins]
+    staticPlugins = []
```

Comment:
I still haven't figured out how to handle this kind of dependency.  It is really only for building the client.  At the moment, the `plugin.json` file is still used by the build system, but it duplicates information added to the entry_point.  It will be nice to consolidate them, but I haven't yet decided how to propagate that information down to grunt.